packages/select/src/utils/getLGSelectTestUtils/getLGSelectTestUtils.ts

```diff
@@ -0,0 +1,160 @@
+    element,
+    `[data-lgid=${LGID_LABEL}]`,
+  );
+
+  /**
+   * Queries the `element` for the description element. Will return `null` if the desription is not found.
+   */
+  const description = queryByQuerySelector<HTMLElement>(
+    element,
+    `[data-lgid=${LGID_DESCRIPTION}]`,
+  );
+
+  /**
+   * Queries the `element` for the input element.
+   */
+  const selectTrigger = queryByQuerySelector<HTMLButtonElement>(
+    element,
+    'button',
+  ) as HTMLButtonElement;
+
+  /**
+   * Queries the `element` for the error message element. Will return `null` if the error message is not found.
+   */
+  const errorMessage = queryByQuerySelector<HTMLElement>(
+    element,
+    `[data-lgid="${LGID_SELECT_ERROR_MESSAGE}"]`,
+  );
+
+  /**
+   * Queries the `element` for aria-disabled.
+   */
+  const isInputDisabled = () => {
+    const ariaDisabled = queryByQuerySelector<HTMLElement>(
+      element,
+      '[aria-disabled="true"]',
+    );
+
+    return !!ariaDisabled;
+  };
+
+  /**
+   * Returns the `Select` value.
+   */
+  const getSelectValue = () => {
+    return selectTrigger.textContent || '';
+  };
+
+  /**
+   * Queries the `element` for the warning Icon.
+   */
+  const isError = () => {
+    const warningIcon = queryByQuerySelector<SVGElement>(
+      element,
+      'svg[title="Error"]',
+    );
+
+    return !!warningIcon;
+  };
+
+  // We cannot query within the select element because if the popover is using a portal, the element will render outside the select element
+  const getPopover = () =>
+    queryByQuerySelector<HTMLDivElement>(
+      document.body,
+      '[data-lgid="lg-select-popover"]',
+    );
+
+  const getAllOptions = (): Array<HTMLLIElement> => {
+    // only one select popover should be open at a time
+    const popover = getPopover();
+
+    if (!popover)
+      throw new Error(
+        'Unable to find an element by: [data-lgid="lg-select-popover"]',
+      );
+
+    // Find all options
+    const allOptions =
+      popover.querySelectorAll<HTMLLIElement>('[role="option"]');
+
+    // unlikley to happen since the select will not render without options
+    if (!allOptions.length)
+      throw new Error('Unable to find any `Select` options.');
+
+    return Array.from(allOptions);
+  };
+
+  const getOptionByValue = (value: string) => {
+    if (!value) throw new Error('Value cannot be empty');
+
+    const allOptions = getAllOptions();
+
+    // Find the option with the value
+    const option = allOptions.find(node => node.textContent === value);
+
+    if (!option) return null;
+
+    return option;
+  };
+
+  const clickOption = (value: string) => {
+    if (!value) throw new Error('Value cannot be empty');
+
+    const option = getOptionByValue(value);
+
+    if (!option)
+      throw new Error(`Unable to find option with the value '${value}'.`);
+
+    // Click option
+    option.click();
+  };
+
+  const clickTrigger = () => {
+    selectTrigger.click();
+  };
+
```

Comment:
The correct way to do this in RTL would be `userEvent.click(getInput());`, or wrap this in `act()`.

When we only call `element.click()` Jest logs the error message:
```
    Warning: An update to Select inside a test was not wrapped in act(...).
    
    When testing, code that causes React state updates should be wrapped into act(...):
    
    act(() => {
      /* fire events that update state */
    });
    /* assert on the output */
```

Reply:
Consumers, I believe, could wrap it themselves with `act` or `waitFor`, but I'll double-check this. I think all the warnings were silent so I missed the warnings.

Reply:
Since we're not tying ourselves to a testing library, I don't think we should be handling interaction within these test utils

Reply:
so get rid of all the `clickOption` and `clickTrigger`? Since we have a util to get the trigger and an individual option, which can be passed to `userEvent` or whatever, getting rid of these shouldn't be an issue. @brookeyalof

Reply:
this is RTL solutioN; https://testing-library.com/docs/user-event/utility/#-selectoptions-deselectoptions

Reply:
the problem with getting rid of clicking is that how would we give you all of the options inside of a select? which i do think that we need to do - see above linked RTL solution that we should mock

Reply:
@TheSonOfThomp what do you suggest?

Reply:
What do you mean by all the options? you could use `getOptionByValue`, which returns an element that you can pass to `userEvent` (Sorry i haven't updated the scope to include each util as yet)

Reply:
Theres also `getOptions` which returns all options 

```
return {
    getLabel: () => getLabel,
    getDescription: () => getDescription,
    getInput: () => getSelectTrigger,
    getErrorMessage: () => getErrorMessage,
    getOptions: () => getAllOptions(),
    getOptionByValue: (value: string) => getOptionByValue(value),
    getPopover: () => getPopover(),
    isDisabled: () => isInputDisabled(),
    isError: () => isError(),
    getInputValue: () => getSelectValue(),
 
    clickOption: (value: string) => clickOption(value),
    clickTrigger: () => clickTrigger(),
  };
```

Reply:
Resolved over Zoom -- we will remove the click utils since clicking may be handled differently in different testing libraries. I'll add this to the scope doc.